<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: none;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="label_income_summary">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>421</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(102, 72, 47);
font: 75 italic 10pt &quot;Arial Bold&quot;;
</string>
    </property>
    <property name="title">
     <string>Monthly Income Summary</string>
    </property>
    <widget class="QLabel" name="label_monthly_income">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(106, 161, 59);
font: 75 italic 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Monthly Income:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_income">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 8px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_deducted">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 8px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_remaining">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(106, 161, 59);
font: 75 italic 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Remaining %</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_total_deducted">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(106, 161, 59);
font: 75 italic 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Total Deducted:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_remaining">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(107, 162, 60);
background-color:  rgb(255, 255, 255);
border-radius: 8px;
border: none;

</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(102, 72, 47);
font: 75 italic 10pt &quot;Arial Bold&quot;;
</string>
    </property>
    <property name="title">
     <string>Expense Tracker</string>
    </property>
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(106, 161, 59);
font: 75 italic 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_expense_name">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 8px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_amount">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>75</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(106, 161, 59);
font: 75 italic 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Amount:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_expense_amount">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>71</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 8px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add_expense">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Bold</family>
       <pointsize>9</pointsize>
       <weight>9</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(106, 161, 59);
font: 75 italic 9pt &quot;Arial Bold&quot;;
</string>
     </property>
     <property name="text">
      <string>Add Expense</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="table_expenses">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>341</width>
       <height>131</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>531</width>
       <height>131</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btn_edit_expense">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(106, 161, 59);
font: 75 italic 9pt &quot;Arial Bold&quot;;
</string>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_delete_expense">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(106, 161, 59);
font: 75 italic 9pt &quot;Arial Bold&quot;;
</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(102, 72, 47);
font: 75 italic 10pt &quot;Arial Bold&quot;;
</string>
    </property>
    <property name="title">
     <string>Progress Summary</string>
    </property>
    <widget class="QLabel" name="label_select_months">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(106, 161, 59);
font: 75 italic 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Select Months for Summary:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinbox_months">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>42</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>42</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 4px;
color: rgb(106, 161, 59);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_summary">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(106, 161, 59);
font: 75 italic 10pt &quot;Arial Bold&quot;;
</string>
     </property>
     <property name="text">
      <string>Summary</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="templabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>801</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(232, 100, 242);
font: 75 italic 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../FM/financepic.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>871</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calisto MT Bold</family>
      <pointsize>30</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(254, 134, 134);
border-radius: 10px;
border: 3px solid  rgb(102, 72, 47);
font: 75 27pt &quot;Cambria&quot;;
font: 75 30pt &quot;Calisto MT Bold&quot;;
border: none;
</string>
    </property>
    <property name="text">
     <string>Finance Manager</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_quote">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>580</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 12pt &quot;Arial Bold&quot;;
color: rgb(179, 133, 76);</string>
    </property>
    <property name="text">
     <string>&quot;Your smart guide to daily budgeting.&quot;</string>
    </property>
   </widget>
   <zorder>templabel</zorder>
   <zorder>label_income_summary</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>label_title</zorder>
   <zorder>label_quote</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
